paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/ResourceRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.paymentsheet.specifications
+
+import com.stripe.android.paymentsheet.address.AddressFieldElementRepository
+
+/**
+ * This holds all the resources read in from JSON.
+ */
+internal class ResourceRepository(
+    internal val bankRepository: BankRepository,
+    internal val addressRepository: AddressFieldElementRepository
+) {
+    internal fun init() {
```

Comment:
Do we need to delay the call where `BankRepository` and `AddressFieldElementRepository` are initialized? 
If not I think this method can be avoided by adding the actual `init {}` block in `BankRepository` and `AddressFieldElementRepository`

Reply:
The init will be doing IO, so I make the init call [here](https://github.com/stripe/stripe-android/blob/f0fb7996c5db7bfc8efc4177d7b9049f111637cc/paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt#L252)